src/NBitcoin/RPC/RPCClient.cs

```diff
@@ -1236,6 +1236,17 @@ public Task SendRawTransactionAsync(byte[] bytes)
 #endregion
 
 #region Utility functions
+        /// <summary>
+        /// Returns information about a base58 or bech32 Bitcoin address
+        /// </summary>
+        /// <param name="address">a Bitcoin Address</param>
+        /// <returns>{ IsValid }</returns>
+        public ValidatedAddress ValidateAddress(BitcoinAddress address)
```

Comment:
You can really just return a boolean here ;)

Reply:
Not really. See my other comment.